Cargo.toml

```diff
@@ -66,7 +66,7 @@ serde = { version = "^1.0.190", optional = true }
 
 [target.'cfg(windows)'.dependencies]
 ntapi = "0.4"
-windows = { version = "0.56", features = [
+windows = { version = ">=0.54, <=0.57", features = [
```

Comment:
```suggestion
# Support a range of versions in order to avoid duplication of this crate. Make sure to test all
# versions when bumping to a new release, and only increase the minimum when absolutely necessary.
windows = { version = ">=0.54, <=0.57", features = [
```

I think a few comments here will help future contributors. :)

Reply:
Indeed! That'd be very welcome. :)